Templates/RemoteRepo/Template/repo.json

```
@@ -0,0 +1,13 @@
+{
+    "repo_name":"${Name}",
+    "repo_uri": "${RepoURI}",
+    "repo_path": "${RepoPath}",
+    "schemaVersion": "1.0.0",
+    "origin":"Origin for the ${Name} Gem Repository",
+    "summary": "${Summary}",
+    "additional_info": "Additional info for ${Name}",
```

If we want to let the user set the value of this field (which we should) then we need to use a unique variable name.
```suggestion
    "additional_info": "${AdditionalInfo}",
```